src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -631,25 +709,25 @@ else if ( NoteLine.contains(CuDisbursementVoucherConstants.DV_EXTRACT_NOTE_PREFI
                                     if (NoteLine.substring(0,2).contains(CuDisbursementVoucherConstants.DV_EXTRACT_TYPED_NOTE_PREFIX_IDENTIFIER)) {
                                         // Trim the first two characters from the note and assign it as the first user typed note line
                                         NoteLine = NoteLine.substring(2);
-                                        FirstNoteAfterAddressInfo = (NoteLine.length() <= 72) ? NoteLine : NoteLine.substring(0,72);
-                                        
+                                        SecondNoteAfterAddressInfo = obtainTruncatedNoteLineSection(NoteLine, FirstNoteAfterAddressInfo);
+                                        FirstNoteAfterAddressInfo = obtainLeadingNoteLineSection(NoteLine, FirstNoteAfterAddressInfo);
```

Comment:
After the call to obtainLeadingNoteLineSection(), it seems that the old value of FirstNoteAfterAddressInfo will be completely discarded, and will not get prepended to the preserved section of the NoteLine. (The same goes for the equivalent processing below for the 2nd line and 3rd line.) Is this intentional?

Reply:
That was not intentional. Good catch. Thank you.